Source/WebInspectorUI/UserInterface/Models/DOMSearchMatchObject.js

```diff
@@ -34,7 +34,8 @@ WI.DOMSearchMatchObject = class DOMSearchMatchObject
         this._domNode = domNode;
         this._title = title;
         this._searchTerm = searchTerm;
-        this._sourceCodeTextRange = resource.createSourceCodeTextRange(textRange);
+        this._textRange = textRange;
```

Comment:
NIT: should we add a `console.assert(textRange instanceof WI.TextRange, textRange)` above?